sotodlib/obs_ops/splits.py

```diff
@@ -55,8 +55,10 @@ def det_splits_relative(aman, det_left_right=False, det_upper_lower=False, det_i
 
     if not(wrap):
         if wrap == True:
+            aman.move('det_flags', None)
```

Comment:
I think that if `det_flags` doesn't exist in aman already this will fail so may need a check like:

```
if name in aman._fields:
    aman.move(name, None)
```